.github/workflows/swift.yml

```diff
@@ -17,7 +17,7 @@ on:
         description: release
 
 env:
-  VERSION: 4.0.0-beta
+  VERSION: 3.10.3
```

Comment:
I think this is a patch update of a previous version. CI should run on the latest version. Or is there any specific reason to use the previous version here?

Reply:
This is just for version numbering into the built SDK.